ui/job-view/headerbars/FiltersMenu.jsx

```diff
@@ -100,26 +98,23 @@ function FiltersMenu(props) {
           <li
             title="Pin all jobs that pass the global filters"
             className="dropdown-item"
-            onClick={pinAllShownJobs}
           >
-            Pin all showing
+            <div onClick={pinAllShownJobs} role="presentation">
+              Pin all showing
+            </div>
           </li>
-          <li
-            title="Show only superseded jobs"
-            className="dropdown-item"
-            onClick={filterModel.setOnlySuperseded}
-          >
-            Superseded only
+          <li title="Show only superseded jobs" className="dropdown-item">
+            <div onClick={filterModel.setOnlySuperseded} role="presentation">
+              Superseded only
+            </div>
           </li>
           <li title={`Show only pushes for ${email}`} className="dropdown-item">
             <a href={getJobsUrl({ author: email })}>My pushes only</a>
           </li>
-          <li
-            title="Reset to default status filters"
-            className="dropdown-item"
-            onClick={filterModel.resetNonFieldFilters}
-          >
-            Reset
+          <li title="Reset to default status filters" className="dropdown-item">
+            <div onClick={filterModel.resetNonFieldFilters} role="presentation">
```

Comment:
I think this div is technically acting like a button and using `role=presentation` to remove semantic value from a div doesn't quite make sense to me. I think what would be better is to replace this dropdown menu with the approproate react-strap dropdown components. You can use the `DropdownMenuItems` component - in our codebase in the ui/shared directory - for the actual dropdown items. This will already give it semantic meaning and a role that should pass this test.

Reply:
@sarah-clements @camd the `DropdownMenuItems` component seems to allow only one selected item at a time. But the FiltersMenu drop down is a richer combination of partly multi-selectable checkbox items and some basic dropdown items. I am not too sure how we can use `DropdownMenuItems` in this context. I may have to change `DropdownMenuItems` to allow multi-select items (which may be a more intrusive change). Any suggestions? :)

Reply:
I think for this situation, don't use the `DropdownMenuItems` component - just use the individual react strap components (and I'd suggest using `UncontrolledDropdown` for simplicity since it doesn't require setting state).